README.md

```diff
@@ -56,6 +56,80 @@ Since this library is used by Home Assistant, see <https://www.home-assistant.io
 
 So follow that advice and try to scrape all fields at once, similar to the `get_form_action_url_and_hidden_inputs` in helpers.py.
 
+## Example Usage - CLI
+
+The opower library comes with a CLI interface to get familiar with the library and read actual data from your utility.
+
+```sh
+# Show information on how to call the library CLI interface
+python -m opower --help
+
+# Example: show recent usage only, for all accounts:
+python -m opower --utility comed --username <username> --password <password> --usage_only --aggregate_type day
```

Comment:
It's bad practice including the password in the command line because it's saved in your bash history so I'd rather not encourage using it in the README. Per my other comment above maybe just remove this line?